Source/RxBluetoothKitLogger.swift

```diff
@@ -0,0 +1,126 @@
+import Foundation
+
+/// Namespace for assigning the default logger.
+public enum RxBluetoothKitLogger {
+    /// The default logger that will be used by RxBluetoothKit
+    public static var defaultLogger: Logger = SimplePrintLogger()
+}
+
+private class SimplePrintLogger: Logger {
+    private var currentLogLevel: RxBluetoothKitLog.LogLevel = .none
+
+    /// Set new log level.
+    /// - Parameter logLevel: New log level to be applied.
+    public func setLogLevel(_ logLevel: RxBluetoothKitLog.LogLevel) {
+        self.currentLogLevel = logLevel
+    }
+
+    /// Get current log level.
+    /// - Returns: Currently set log level.
+    public func getLogLevel() -> RxBluetoothKitLog.LogLevel {
+        return currentLogLevel
+    }
+
+    func log(
+        _ message: @autoclosure () -> String,
+        level: RxBluetoothKitLog.LogLevel,
+        file: StaticString,
+        function: StaticString,
+        line: UInt
+    ) {
+        log(
+            message(),
+            level: level,
+            file: String(describing: file),
+            function: String(describing: function),
+            line: line
+        )
+    }
+
+    func log(
+        _ message: @autoclosure () -> String,
+        level: RxBluetoothKitLog.LogLevel,
+        file: String,
+        function: String,
+        line: UInt
+    ) {
+        if currentLogLevel <= level {
+            print("\(tag(with: level)) \(message())")
```

Comment:
Now when you add information about file, function and line, we can use this information in this standard logger and print it as well.